### PR TITLE
Increased scheduler capactiy, recurion tip arity, and new verification keys.

### PR DESCRIPTION
**Pull request by mm-zk**
# What ❔

* Increased scheduler capacity to 34100 circuits.
* Increased arity (capacity) of recursion tip to 32 (this was done in zkevm_circuits)
* generated new verification keys for this setup.

### CODE REVIEW

File: circuit_definitions/src/circuit_definitions/recursion_layer/mod.rs [hunk 1]
```diff
@@ -27,7 +27,7 @@ pub const RECURSION_ARITY: usize = 32;
 // Maximum amount of basic circuits that a scheduler can handle.
 // The value was selected in such a way, that the scheduler circuit
 // fits into 2^20 trace size (currently it uses aroudn 1'043'000)
-pub const SCHEDULER_CAPACITY: usize = 24100;
+pub const SCHEDULER_CAPACITY: usize = 34100;
```
Comment (EmilLuta) [inline circuit_definitions/src/circuit_definitions/recursion_layer/mod.rs:34]: Very arbitrary number. Can we add some more information on the choice?

Comment (mm-zk) [reply circuit_definitions/src/circuit_definitions/recursion_layer/mod.rs:34]: Added more info.

Comment (EmilLuta) [reply circuit_definitions/src/circuit_definitions/recursion_layer/mod.rs:34]: Not sure where, to be honest. Maybe the env is not loading the changes for me?

Comment (mm-zk) [reply circuit_definitions/src/circuit_definitions/recursion_layer/mod.rs:34]: try now.

**Pull request merged**